eqllib/analytics/defense-evasion/T1223-compiled-html-file.toml

```
@@ -0,0 +1,19 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "medium"
+contributors = ["Dan Beavin"]
+created_date = "08/08/2019"
+description = "Identifies usage of hh.exe when executing .chm files."
+id = "b25aa548-7937-11e9-8f5c-d46d6d62a49e"
+name = "HH.exe execution"
+os = ["windows"]
+tactics = ["Compiled HTML File"]
```

Looks good. Only other change is to update this to the ATT&CK tactic "Defense Evasion"
```suggestion
tactics = ["Compiled HTML File"]
```